src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfExpressionBuilder.cs

```diff
@@ -162,7 +162,10 @@ public static int DwarfRegNum(TargetArchitecture architecture, int regNum)
                     };
 
                 case TargetArchitecture.LoongArch64:
+                    Debug.Assert(regNum <= 64);
```

Comment:
`using System.Diagnostics;` at the top of this file is needed.

Reply:
Oh, I forgot it. So should I delete this line or add `using System.Diagnostics` at the top?
Thanks for your help!

Reply:
Add after https://github.com/dotnet/runtime/blob/926885adc308007934a97e6d36b59cfb7a535a66/src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfExpressionBuilder.cs#L5 (so they are alphabetized)

Reply:
OK, thanks a lot!